sources/packages/backend/apps/api/src/services/student-assessment/student-assessment.service.ts

```diff
@@ -211,28 +200,8 @@ export class StudentAssessmentService extends RecordDataModelService<StudentAsse
         "studentScholasticStanding.id",
         "application.id",
         "applicationException.id",
+        "assessment.studentAssessmentStatus as status",
```

Comment:
as is not needed  anymore as we are returning the status as is right?

connected to https://github.com/bcgov/SIMS/pull/2588/files#r1424700168

Reply:
`as` was needed. but the query now adjusted as part of https://github.com/bcgov/SIMS/pull/2588#discussion_r1424700168

Reply:
Precisely I was saying that with `getRawAndEntities()` being changed to `getMany()` or any non raw result `as` shouldn't be required anymore.